apps/frontend/src/pages/access-management/libs/components/groups-table/groups-table.tsx

```diff
@@ -28,20 +29,48 @@ const GroupsTable = ({
 	paginationBackground = "primary",
 	totalItemsCount,
 }: Properties): JSX.Element => {
-	const handleEdit = (groupId: number): void => {
-		const group = groups.find(({ id }) => id === groupId);
-		onEdit(group as GroupGetAllItemResponseDto);
-	};
-
-	const groupColumns = getGroupColumns({
-		onDelete,
-		onEdit: handleEdit,
-	});
+	const handleEdit = useCallback(
+		(groupId: number): void => {
+			const group = groups.find(({ id }) => id === groupId);
+			const hasGroup = Boolean(group);
+
+			if (hasGroup) {
+				onEdit(group as GroupGetAllItemResponseDto);
```

Comment:
I think we can omit this constant, can't we?
```suggestion
			if (group) {
				onEdit(group as GroupGetAllItemResponseDto);
```

Reply:
I thought that the variable name provided more clarity. But yes, you're right, I think it’s even better to write it like this:
```
if (group) {
	onEdit(group);
}
```